src/test/java/core/basesyntax/SalaryInfoTest.java

```diff
@@ -81,8 +81,8 @@ public class SalaryInfoTest {
 
     @Test
     public void getSalaryInfoByMonth() {
-        SalaryInfo salary = new SalaryInfo();
         for (int i = 0; i < dates.length; i++) {
+            SalaryInfo salary = new SalaryInfo();
```

Comment:
NEVER CHANGE TEST

revert changes in tests.

Reply:
accepted, sorry for that, changed back